frontend/src/components/search/SearchResultsList.tsx

```diff
@@ -40,7 +40,7 @@ export default async function SearchResultsList({
   }
 
   return (
-    <ul className="usa-list--unstyled">
+    <ul className="usa-list--unstyled" data-testid="search-list">
```

Comment:
is there a way to query for this without adding a testid? Sometimes with testing-library the limitations they put on their queries make test ids necessary but since playwright lets you do arbitrary DOM queries I feel like it shouldn't be necessary to add a test id here.